data/tlg2018/tlg002/tlg2018.tlg002.1st1K-eng1.xml

```diff
@@ -1,133 +1,17538 @@
+                        </editor>
+                        <author ref="urn:cts:greekLit:tlg2018">Eusebius</author>
+                        <imprint>
+                            <publisher>William Heinemann</publisher>
+                            <publisher>G.P. Putnam's Sons</publisher>
+                            <pubPlace>New York</pubPlace>
+                            <pubPlace>London</pubPlace>
+                            <date>1926-1932</date>
+                        </imprint>
+                        <biblScope unit="volume">1-2</biblScope>
+                    </monogr>
+                    <ref target="https://archive.org/details/ecclesiasticalhi01euseuoft/">Internet Archive</ref>
+                    <ref target="https://archive.org/details/ecclesiasticalhi02euseuoft">Internet Archive</ref>
+                </biblStruct>
+            </listBibl>
+        </sourceDesc>
     </fileDesc>
     <encodingDesc>
-      <p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
-      <refsDecl n="CTS">
-        <cRefPattern n="paragraph" matchPattern="(.+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"/>
-      </refsDecl>
+        <p>Text encoded in accordance with the latest EpiDoc standards (January 2014)</p>
+          <refsDecl n="CTS">
+            <cRefPattern n="chapter" matchPattern="(.+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1']/tei:div[@n='$2'])"/>
+            <cRefPattern n="book" matchPattern="(.+)" replacementPattern="#xpath(/tei:TEI/tei:text/tei:body/tei:div/tei:div[@n='$1'])"/>
+        </refsDecl>
     </encodingDesc>
     <profileDesc>
-      <langUsage>
-        <language ident="grc">Greek</language>
-        <language ident="lat">Latin</language>
-      </langUsage>
+        <langUsage>
+            <language ident="eng">English</language>
+            <language ident="lat">Latin</language>
+        </langUsage>
     </profileDesc>
     <revisionDesc>
-      <change/>
+        <change/>
     </revisionDesc>
-  </teiHeader>
-  <text>
-    <body>
-      
-      <div type="commentary" subtype="preface" xml:lang="eng" n="urn:cts:greekLit:tlg2018.tlg002.1st1K-eng1">
-        <pb n="v"/>
-        <head>PREFACE</head>
-        <div type="textpart" subtype="paragraph" n="1"><p>THE text of the Historia Ecclesiastica of Eusebius in
-          the following pages is that of E. Schwartz in volumes
-          i and ii. of Part II. of the edition of Eusebius in
-          Die griechischen christlichen Schriftsteller der erstern
-          drei Gahrhunderte, published by the Berlin Academy.
-          Changes have only been introduced in a few very
-          small points, such as the accentuation of the proper
-          names in the genealogy of Christ.</p>
+</teiHeader>
+    <text>
+        <body>
+            <div type="translation" xml:lang="eng" n="urn:cts:greekLit:tlg2018.tlg002.opp-eng1">
```

Comment:
Wrong file name.

Reply:
Thanks, thought I caught all of the changes.